pre_commit_hooks/requirements_txt_fixer.py

```diff
@@ -94,7 +99,7 @@ def fix_requirements(f):  # type: (IO[bytes]) -> int
 
     after_string = b''.join(after)
 
-    if before_string == after_string:
+    if before_string.rstrip() == after_string.rstrip():
```

Comment:
I would just leave this line alone personally -- I think adding a newline to the end of the file is a good fix in the general case 👍

Reply:
will do, but as a code design question, why not to run the `end_of_file_fixer` at first as is?

Reply:
Some prefer to pick and choose hooks, I agree someone should probably be configuring the end-of-file-fixer as one of their default hooks -- but it's probably ok to have the behaviour happen here as well